src/components/base/outline-tabs/outline-tab-group/outline-tab-group.ts

```diff
@@ -60,7 +60,7 @@ export default class OutlineTabGroup extends OutlineElement {
     window.addEventListener('resize', this._handleResize);
```

Comment:
We should not use addEventListener with `resize` (old method), it's very costly for performance.
That's what `ResizeObserver` is for (new method)